src/main/java/net/dv8tion/jda/api/entities/User.java

```diff
@@ -229,29 +229,36 @@ default String getEffectiveAvatarUrl()
     /**
      * Represents the bit offsets used by Discord for public flags
      */
-    enum Flags{
-        NONE(                      0, "None"),
-        STAFF(                     1, "Discord Employee"),
-        PARTNER(                   2, "Discord Partner"),
-        HYPESQUAD(                 4, "HypeSquad Events"),
-        BUG_HUNTER_LEVEL_1(        8, "Bug Hunter Level 1"),
-        // Bravery
-        HYPESQUAD_ONLINE_HOUSE_1( 64, "HypeSquad Online House Bravery"),
-        // Brilliance
-        HYPESQUAD_ONLINE_HOUSE_2(128, "HypeSquad Online House Brilliance"),
-        // Balance
-        HYPESQUAD_ONLINE_HOUSE_3(256, "HypeSquad Online House Balance"),
-        PREMIUM_EARLY_SUPPORTER( 512, "Early Supporter"),
-        TEAM_USER(              1024, "Team User"),
-        SYSTEM(                 4096, "System User"),
-        BUG_HUNTER_LEVEL_2(    16384, "Bug Hunter Level 2"),
-        VERIFIED_BOT(          65536, "Verified Bot"),
-        VERIFIED_DEVELOPER(   131072, "Verified Developer");
-        
+    enum Flags {
+        NONE(                         "None"),
+        STAFF(                     0, "Discord Employee"),
+        PARTNER(                   1, "Discord Partner"),
+        HYPESQUAD(                 2, "HypeSquad Events"),
+        BUG_HUNTER_LEVEL_1(        3, "Bug Hunter Level 1"),
+
+        // HypeSquad
+        HYPESQUAD_BRAVERY(         6, "HypeSquad Bravery"),
+        HYPESQUAD_BRILLIANCE(      7, "HypeSquad Brilliance"),
+        HYPESQUAD_BALANCE(         8, "HypeSquad Balance"),
+
+        EARLY_SUPPORTER(           9, "Early Supporter"),
+        TEAM_USER(                10, "Team User"),
+        SYSTEM(                   12, "System User"),
+        BUG_HUNTER_LEVEL_2(       14, "Bug Hunter Level 2"),
+        VERIFIED_BOT(             16, "Verified Bot"),
+        VERIFIED_DEVELOPER(       17, "Verified Bot Developer");
+
         private final int offset;
         private final long raw;
         private final String name;
-        
+
+        Flags(@Nonnull String name)
+        {
+            this.offset = 0;
```

Comment:
This.... makes no sense to keep at 0.

Reply:
The value itself should be 0, once again [as per the docs](https://discordapp.com/developers/docs/resources/user#user-object-user-flags); If offset is changed from final and not used in the constructor it will default to 0.

Reply:
Take a look at the Permission enum.
It should be based of this and I think you get something clearly wrong here.